src/controllers/TaskController.ts

```diff
@@ -0,0 +1,129 @@
+import { Request, Response } from "express";
+import { getRepository } from "typeorm";
+import { validate } from "class-validator";
+ 
+import { Task } from "../entities/Task";
+import { Project } from "../entities/Project";
+
+class TaskController {
+  static getAll = async (req: Request, res: Response) => {
+    const projectId: number = req.params.projectId;
```

Comment:
Here we are getting the `projectId` from the request params, but we are not using it. And more importantly we are not checking if the `project` existing with this provided `projectId`.